sign_node/signer.py

```diff
@@ -179,6 +181,8 @@ def check(pkg_path: str) -> typing.Tuple[SignStatusEnum, str]:
                 sig = signature.signer.lower()
                 if sig == key_id_lower:
                     return SignStatusEnum.SUCCESS, ''
+                elif subkeys and sig in subkeys:
```

Comment:
we can unite two if conditions in one and return `success` only once
also we can check subkeys like this `sig in subkeys`